tests/basic/misc_test.go

```diff
@@ -1784,4 +1784,62 @@ var _ = Describe("{KubeClusterRestart}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
-})
\ No newline at end of file
+})
+
+var _ = Describe("{VerifyNoPxRestartDueToPxPodStop}", func() {
+	JustBeforeEach(func() {
+		var testrailID = 87311294
+		StartTorpedoTest("VerifyNoPxRestartDueToPxPodStop", "Verify that px serivce remain up even if px pod got deleted ", nil, 0)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	It("has to setup, validate and teardown apps", func() {
+
+		// Get uptime for px service on each node
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			processPid := make(map[string]string)
+			startCmd := "pidof px"
+			//Capturing PID pf PX before stopping PX pods
+			for _, nnode := range node.GetStorageNodes() {
+				output, _ := Inst().N.RunCommand(nnode, startCmd, node.ConnectionOpts{
+					Timeout:         30 * time.Second,
+					TimeBeforeRetry: 20 * time.Second,
+					Sudo:            true,
+				})
+				processPid[nnode.Id] = output
+			}
+			log.Infof(fmt.Sprintf("Process IDs for px before stopping portworx pod  %s", processPid))
+
+			//Deleting px pods from all the node
+			err = DeletePXPods("kube-system")
```

Comment:
verify err once DeletePxPods are done 

also not every time px will be installed in kube-system namespace , we should make sure to pick the namespace where px pods are running